onnxruntime/python/tools/transformers/models/whisper/convert_to_onnx.py

```diff
@@ -378,6 +373,31 @@ def main(argv=None):
         chain_model(args)
         output_paths.append(args.beam_model_output_dir)
 
+        # Check chained model
+        ort_session = create_onnxruntime_session(
+            args.beam_model_output_dir,
+            use_gpu=args.use_gpu,
+            provider=["CUDAExecutionProvider", "CPUExecutionProvider"] if args.use_gpu else ["CPUExecutionProvider"],
+        )
+        device = torch.device("cuda:0" if args.use_gpu else "cpu")
+
+        # Wrap parity check in try-except to allow export to continue in case this produces an error
+        try:
+            with torch.no_grad():
+                max_diff = WhisperHelper.verify_onnx(args.model_name_or_path, ort_session, device)
+            logger.info(f"Max difference between PyTorch and ONNX Runtime results = {max_diff}")
+            if max_diff > 1e-4:
+                logger.warning("PyTorch and ONNX Runtime results are NOT close")
+        except Exception as e:
+            logger.warning("An error occurred while trying to verify parity between PyTorch and ONNX Runtime.")
+            logger.warning(e)
```

Comment:
merge two warnings into one like 
logger.warning(f"An error {e} occurred while trying to verify parity between PyTorch and ONNX Runtime.")

Reply:
Fixed